src/app/category-chart/trendline/category-chart-with-trendline.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="sample-wrapper container">
+    <div class="chart">
+        <igx-category-chart height="100%" width="100%" #chart [dataSource]="data" chartTitle="Olympic Medals By Country" chartType="column"
+            trendLineType="linearFit" [tooltipTemplate]="valueTooltip" (seriesAdded)="seriesAddedEvent($event)">
+        </igx-category-chart>
+    </div>
+
+    <ng-template let-series="series" let-item="item" #valueTooltip>
+        <div class="tooltipTitle">
+            <span>{{item.Year}} :
+                <label [style.color]="series.actualBrush" [textContent]="series.name"></label>
+                <div *ngIf="series.name == 'USA'">
+                    <img src="assets/images/grid/flags/us.png" style="width:40px;height:30px;border:1px solid black;">
+                </div>
+                <div *ngIf="series.name == 'China'">
+                    <img src="assets/images/grid/flags/cn.png" style="width:40px;height:30px;border:1px solid black;">
+                </div>
+                <div *ngIf="series.name == 'Russia'">
+                    <img src="assets/images/grid/flags/ru.png" style="width:40px;height:30px;border:1px solid black;">
+                </div>
+            </span>
+        </div>
```

Comment:
@agoldenbaum are you showing all 3 flags in one tooltip? If so, can you just show on flag by binding image's source to a new column (image) on data item?